src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -95,7 +95,8 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 };
 
 const webhookRouter = async (context: WebhookContext) => {
-	const VALID_PULL_REQUEST_ACTIONS = ["opened", "reopened", "closed", "edited", "review_requested"];
+	const VALID_PULL_REQUEST_ACTIONS = ["opened", "reopened", "closed", "edited", "converted_to_draft", "ready_for_review"];
```

Comment:
Thanks for the PR! Was it intentional to omit the `review_requested` action? This could be useful for triggering issue transitions, e.g. to "In Review".